letsencrypt-nginx/letsencrypt_nginx/configurator.py

```diff
@@ -136,7 +136,14 @@ def deploy_cert(self, domain, cert_path, key_path,
 
         .. note:: This doesn't save the config files!
 
+        :raises errors.PluginError: When unable to deploy certificate due to
+            a lack of directives or configuration
+
         """
+        if not fullchain_path:
+            raise errors.PluginError(
+                "--fullchain-path is required for nginx plugin.")
```

Comment:
"The nginx plugin currently requires --fullchain-path to install a cert".  That requirement will go away when we resolve #1391; in most cases `letsencrypt install` should be able to find previously obtained certs.